Meta/configure-clangd.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+function usage {
+    echo "$0 <.clangd file path> [release|debug]"
+    echo "Update local clangd configuration with the proper"
+    echo "compilation database according to the selected build type."
+}
+
+script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+cd "${script_path}/.." || exit 1
+
+# Check if the user has sed.
+
+if ! which sed >/dev/null 2>&1; then
+    echo "Error: No sed found. Cannot configure .clangd automatically."
+    exit 1
+fi
+
+# Check if the user specified the right number of parameters.
+if [ $# -ne 2 ]; then
+    usage 
+    exit 1
+fi
+
+clangd_file_path=$1
+if [ ! -f "$clangd_file_path" ]; then
+    echo "Error: ${clangd_file_path} is not a regular file."
+    echo
+    usage
+    exit 1
+fi
+
+build_type=""
+case $2 in
+    debug)
+        build_type="-$2"
+        ;;
+    release)
+        build_type=""
+        ;;
+    *)
+        echo "Invalid build configuration specified: $2"
+        usage
+        exit 1
+esac
```

Comment:
Hm, these don't quite align with the build types and directories set in CMakePresets.json.

Those being:

default (RelWithDebInfo) : Build/ladybird
Debug (Debug): Build/ladybird-debug
Sanitizer: Build/ladybird-sanitizers

That being said, we are missing a `build` and `test` preset for the `Debug` configure preset.

I will note that in this project basically none of the regular contributors are familiar with or use the multi-config setups common in Windows or Xcode centric environments, or Ninja multi-config.

Is it common to want to switch your clangd configuration to match either release or debug? Personally I just always have a `default` build up to date, and build debug or sanitizers as needed 🤔